packages/odyssey-react-mui/src/theme/index.ts

```diff
@@ -10,5 +10,8 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-export { theme as odysseyTheme } from "./theme";
+export * from "./theme";
 export { useTheme } from "./useTheme";
+export interface TokenOverrideOptions {
+  [k: string]: string | number;
+}
```

Comment:
I don't like using `interface` over `type`. But we could also make this simpler:
```ts
export type TokensOverride = Record<string, string | number>
```

But even still, this isn't specific enough. It would be good to define the type of the override as the token itself:
```ts
export type TokensOverride = typeof Tokens
```

Reply:
from Kevin.. `export type TokensOverride = Partial<typeof Tokens>`